code/script.js

```diff
@@ -19,29 +47,173 @@ const showMessage = (message, sender) => {
   } else if (sender === 'bot') {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/Pizzabot.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
 // Starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hello, what's your name?`, 'bot') 
 }
 
-// Set up your eventlisteners here
-
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
 setTimeout(greeting, 1000)
+
+// Question 1
+const handleNameInput = (event) => {
+  event.preventDefault()
+  const userName = nameInput.value
+  showMessage(userName, 'user')
+  nameInput.value = ''
+  setTimeout (() => showMessage(`Nice to meet you ${userName}! I hope you're hungry`, 'bot'), 1000 )
+  setTimeout(() => showMessage('What would you like to eat?', 'bot'), 2000)
+  setTimeout (() => handleFoodInput(userName), 2000)
+}
+
+nameForm.addEventListener('submit', handleNameInput)
+
+// Question 2
+const handleFoodInput = () => {
+  inputWrapper.innerHTML = `
+    <button id="Pizza-button">Pizza</button>
+    <button id="Pasta-button">Pasta</button>
+    <button id="Salad-button">Salad</button>
+  `
+  
+  document
+  .getElementById('Pizza-button')
+  .addEventListener('click',() => {
+    showMessage('I want pizza', 'user')
+    inputWrapper.innerHTML = '' 
+    setTimeout(() => askFoodChoice('pizza'), 1000)
+  })
```

Comment:
again here we have some opportunity to build a function to store repeated code 🤓